guide/how-it-works/ecash/overview.md

```diff
@@ -0,0 +1,222 @@
+   height = 456
+   layout = "full-width"
+%}
+
+The process of creating ecash on Bitcoin is known as **minting.** The process of redeeming ecash into sats via the Lightning Network is known as **melting.** This process typically includes the following steps:
+
+#### Minting ecash
+
+**1. User Deposits Funds** - The user deposits Bitcoin into a mint. Typically this is done by generating a LN invoice through the mint and paying it.
+
+**2. Blinded Token Creation** - Upon successful payment, the mint generates blinded tokens, which are cryptographic representations of the deposited funds. The blinding process ensures that the mint cannot link the tokens to the user, preserving privacy.
+
+**3. Mint provides Tokens to User** - The user receives a set of tokens that can be combined to create an ecash token of any amount denominated in Bitcoin.
+
+#### Melting ecash
+
+**4. User Initiates Melting** - The user requests to redeem their ecash tokens for Bitcoin through the Lightning Network.
+
+**5. Token destruction** - The mint verifies and destroys the ecash tokens, ensuring they cannot be reused. This step confirms the authenticity of the tokens and maintains the integrity of the system.
+
+**6. Sats Transfer via Lightning Network** - The user receives an equivalent amount of Bitcoin, completing the melting process and converting ecash back into spendable Bitcoin.
+
+### Sending and receiving ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of ecash is that users are simply transacting pieces of data. These pieces of data are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
+
+#### Text message, such as email or SMS:
+Users can send ecash by copying and pasting the token string. This is particularly useful for online transactions or when using text-based communication platforms.
+
+#### Bluetooth transmission:
+Most applications allow users to share a token via Bluetooth. However, more comprehensive Bluetooth sending and receiving functionalities have not yet been fully developed. As a result, when a user attempts to share an ecash token via Bluetooth, the most common outcome is that the receiving device does not know how to handle it.
+
+#### QR codes:
```

Comment:
QR codes also work for the base chain and lightning, which does not fit with the statement above about unique ways ecash can be spent.

Reply:
As far as I know base chain and lightning QRs are just pointers for where to send the bitcoin. I don't think there is a way to embedded bitcoin in a way that it can claimed just by scanning a QR, right?  What i'm trying to explain in the QR codes section is that the ecash token text string can be embedded into a QR code which a user can claim by simply scanning the QR. I can update that paragraph to better explain it

Update:
Unlike base chain and Lightning Network QR codes, which only provide directions for where to send Bitcoin, ecash tokens can be embedded within a QR code itself. This means that a user can claim the ecash token by simply scanning the QR code, eliminating the need for additional steps. This method is particularly useful for in-person transactions or quick transfers. QR redemption also enables physical bearer assets like paper notes or coins.

Reply:
That's right. For the bitcoin network, it's an address. And for lightning, it's typically an invoice or a pointer to a node (for getting an invoice).

> eliminating the need for additional steps

Would be good to be more clear about those steps. Are those technical steps or user interactions? Technically, the claimant still needs to be online to redeem the ecash token at the mint, right?

Quite tricky to explain this clearly and precisely...

Reply:
I think we can remove this sentence. It's not really 'extra steps' the ability to scan a QR code and claim the funds is just something that ecash can do that on chain and lightning can't.

Update
`Unlike base chain and Lightning Network QR codes, which only provide directions for where to send Bitcoin, ecash tokens can be embedded within a QR code itself. This means that a user can claim the ecash token by simply scanning the QR code. This method is particularly useful for in-person transactions or quick transfers. QR redemption also enables physical bearer assets like paper notes or coins.`

Yes, technically the claimant needs to be online to redeem the ecash token with the mint. However, some wallets allow users to receive or claim ecash tokens while offline. In such cases, the wallet displays a warning that it is currently offline and informs the user that the token will be claimed once the wallet is back online. The wallet then holds the token in a queue, where it usually appears in the transaction history as ‘pending’ or ‘received offline.’ Once the wallet is back online, it attempts to redeem the ecash.

There are risks with this approach. The most obvious is that while the token hasn’t been redeemed, it could technically be double-spent. If the sender sends that token to someone else or claims it themselves before the claimant, it will become invalid.